workspace/find-packages/test/__fixtures__/suppress-warning-for-non-root-project/packages/foo/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "foo",
+  "version": "1.0.0",
+  "pnpm": {
+    "overrides": {},
+    "resolutions": {}
+  },
+  "resolutions": {}
+}
```

Comment:
Side effect is that empty fields no longer warn _(should be fine since nothing is read from them)_.